src/main/java/org/example/daos/DeliveryEmployeeDao.java

```diff
@@ -0,0 +1,46 @@
+package org.example.daos;
+
+import org.example.models.DeliveryEmployee;
+import org.example.models.DeliveryEmployeeRequest;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DeliveryEmployeeDao {
+
+    public int createDeliveryEmployee(DeliveryEmployeeRequest deliveryEmployee) throws SQLException {
+
+        Connection c = DatabaseConnector.getConnection();
+
+        String insertStatement = "INSERT INTO employee (name, salary, bankNumber, nationalInsuranceNo) VALUES (?,?,?,?);";
```

Comment:
How can we specify if an employee is a delivery employee? Should there be measures to ensure that?